content/applications/finance/accounting/payables/misc/forecast.rst

```diff
@@ -2,71 +2,49 @@
 Forecast future bills to pay
 ============================
 
-When you get hundreds of vendor bills per month with each of them having
-different payment terms, it could be complex to follow what you have to
-pay and when. Paying your vendors too early can decrease your cash
-availabilities and paying too late can lead to extra charges.
-
-Fortunately, Odoo provides you the right tools to manage payment orders
-to vendors efficiently.
+In Odoo you have the possibility to manage payments by setting automatic **Payments Terms**
+and **follow-ups**.
 
 Configuration: payment terms
 ============================
 
-In order to track the vendor conditions, we use payment terms in Odoo.
-Payment terms allow to keep track of the conditions to compute the due
-date on an invoice. As an example, a payment term can be:
+In order to track the vendor conditions, we use **Payment Terms** in Odoo.
+**Payment terms** allow keeping track of due dates on invoices. Examples of **Payment Terms** are:
 
 -  50% within 30 days
-
 -  50% within 45 days
 
-To create your most common payment terms, use the menu: :menuselection:`Configuration -->
-Management --> Payment Terms` in the **Accounting** application. The following
-example show a payment term of 30% directly and the balance after 30
-days.
+To create **Payment terms**, use the :menuselection:`Accounting app --> Configuration -->
+Invoicing --> Payment Terms`. By clicking on :guilabel:`Create`, you can add the terms you need as
+well as modify existing ones by clicking on it in the Payment Terms list in the same view.
 
-.. image:: forecast/forecast01.png
-  :align: center
+.. seealso::
+   - `Odoo Tutorials: Payment terms <https://www.odoo.com/slides/slide/payment-terms-1679?fullscreen=1>`_
 
-Once payment terms are defined, you can assign them to your vendor by
-default. Set the Vendor Payment Term field on the Accounting tab of a
-vendor form. That way, every time you will purchase to this vendor, Odoo
-will propose you automatically the right payment term.
-
-.. image:: forecast/forecast02.png
-  :align: center
+Once **Payment Terms** are defined, you can assign them to your vendor by default.
+To do so go to :menuselection:`Vendors --> Vendors` choose the :guilabel:`Vendor` you want and in
+the :guilabel:`Sales & Purchase` tab you can select a specific **Payment Term**.
```

Comment:
Can you add a "," after To do so and after vendors? 
Also, I'd take the "you can" off to put an imperative instead; here's the suggestion
```suggestion
To do so, go to :menuselection:`Vendors --> Vendors`, choose the :guilabel:`Vendor` you want, and, in
the :guilabel:`Sales & Purchase` tab, select a specific **Payment Term**.
```